docs/stdlib/datetime.rst

```diff
@@ -101,35 +101,34 @@ Dates and Times
 
 .. _ref_std_datetime_intro:
 
-EdgeDB has two classes of date/time types:
+In EdgeDB, there are two types of date/time constructs:
 
 * a timezone-aware :eql:type:`std::datetime` type;
 
-* a set of "local" date/time objects, not attached to any particular
-  timezone: :eql:type:`cal::local_datetime`, :eql:type:`cal::local_date`,
-  and :eql:type:`cal::local_time`.
+* a set of "local" date/time types, not attached to any particular
+  timezone, such as :eql:type:`cal::local_datetime`,
```

Comment:
```suggestion
  timezone: :eql:type:`cal::local_datetime`, :eql:type:`cal::local_date`, and
```

I believe this is an exhaustive list. "such as" implies these are examples plucked from a larger list.